### PR TITLE
question-on-arrays

### PR DESCRIPTION
**Pull request by kashif-raza2019**
Array question first solved 11 th and then 26th

### CODE REVIEW

File: Kashif/11.Container-With-Most-Water.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+public:
+    int maxArea(vector<int>& height) {
+         int i=0;
+         int j=height.size()-1;
+         int maxArea=INT_MIN;
+            while(j>=i)
+            {
+                int minHeight=min(height[i],height[j]);
+                int width=j-i; 
+                int currArea=minHeight*width;   //area=height * width
```
Comment (AdityaPrakash-26) [inline Kashif/11.Container-With-Most-Water.cpp]:
```suggestion
                int currArea=minHeight*width;    //area=height * width
```

File: Kashif/11.Container-With-Most-Water.cpp [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                maxArea=max(maxArea,currArea);
+                if(minHeight==height[i]) //if i arr[i] is smaller we increment the width
```
Comment (AdityaPrakash-26) [inline Kashif/11.Container-With-Most-Water.cpp]:
```suggestion
                if(minHeight==height[i])    //if i arr[i] is smaller we increment the width
```

File: Kashif/26.Remove-Duplicates-from-Sorted-Array.cpp [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    
+    /*
+      Runtime: 8 ms, faster than 82.68% of C++ online submissions for Remove 
+      Duplicates from Sorted Array.
+      
+      Memory Usage: 13.8 MB, less than 18.53% of C++ online submissions for Remove                 Duplicates from Sorted Array.
+    
+       Solution done with the help of STL function unique
+       which returns all the unique elements from the vector 
+    */
```
Comment (AdityaPrakash-26) [inline Kashif/26.Remove-Duplicates-from-Sorted-Array.cpp:9]: I suggest moving this comment to the top of the file

File: Kashif/26.Remove-Duplicates-from-Sorted-Array.cpp [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    int removeDuplicates(vector<int>& nums) {
+        if(nums.size() == 0){//Case when length == 0 return 0
```
Comment (AdityaPrakash-26) [inline Kashif/26.Remove-Duplicates-from-Sorted-Array.cpp]:
```suggestion
        if(nums.size() == 0){    //Case when length == 0 return 0
```

File: Kashif/7.Reverse-Integer.cpp [hunk 5]
```diff
@@ -0,0 +1,23 @@
+/*   
+     Runtime: 4 ms, faster than 39.45% of C++ online submissions for Reverse Integer.
+     Memory Usage: 5.8 MB, less than 72.03% of C++ online submissions for Reverse Integer. 
+*/
+
+//Previous solution has runtime error
```
Comment (AdityaPrakash-26) [inline Kashif/7.Reverse-Integer.cpp]:
```suggestion
// Previous solution had runtime error 
```

File: Kashif/6.zig-zag-conversion.cpp [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+ * [6] ZigZag Conversion
+ */
+
+
+/*
+    Accepted
+    1157/1157 cases passed (8 ms)
+    Your runtime beats 86.7 % of cpp submissions
+    Your memory usage beats 84.37 % of cpp submissions (8.2 MB)
+*/
```
Comment (AdityaPrakash-26) [inline Kashif/6.zig-zag-conversion.cpp:6]: I suggest moving this to the top of the file

File: Kashif/7.Reverse-Integer.cpp [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+        {
+        r=temp%10;
+        if (a > INT_MAX/10 || (a == INT_MAX / 10 && r > 7)) return 0;
+        if (a < INT_MIN/10 || (a == INT_MIN / 10 && r < -8)) return 0;
+        a=a*10+r;
+        temp/=10;
+        }
+        return a;
+    }
+
```
Comment (AdityaPrakash-26) [inline Kashif/7.Reverse-Integer.cpp]:
```suggestion
```

File: Kashif/7.Reverse-Integer.cpp [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+//Previous solution has runtime error 
+class Solution {
+public:
+    int reverse(int x) {
+        int temp,a=0,r;
+        temp=x;
+        while(temp!=0)
+        {
+        r=temp%10;
```
Comment (AdityaPrakash-26) [inline Kashif/7.Reverse-Integer.cpp]: The code within the while loop should be properly indented

**Review by AdityaPrakash-26 — CHANGES_REQUESTED** [review]

File: Kashif/7.Reverse-Integer.cpp (hunk 5 above)
Comment (kashif-raza2019) [reply Kashif/7.Reverse-Integer.cpp]: Arey yaar! ab english bhi dekh rha hai

File: Kashif/26.Remove-Duplicates-from-Sorted-Array.cpp (hunk 3 above)
Comment (kashif-raza2019) [reply Kashif/26.Remove-Duplicates-from-Sorted-Array.cpp:9]: Done!

**Review by kashif-raza2019 — COMMENTED** [review]
I have reviewed your suggestions

**Comment by kashif-raza2019** [comment]
Hope it's okay now!

**Comment by AdityaPrakash-26** [comment]
Yep! Thanks for the contribution :D

**Pull request merged**